subsys/bluetooth/mesh/shell_utils.c

```diff
@@ -0,0 +1,155 @@
+/*
+ * Copyright (c) 2022 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#include <shell/shell.h>
```

Comment:
Missing `#include "shell_utils.h"`

Reply:
I'm not quite sure if I understand. You mean `#include <bluetooth/mesh/shell.h>`?

Reply:
Yes, sorry, `bluetooth/mesh/shell.h`.